### PR TITLE
SW-2415 / SW-2416 Visual Updates for Add/EditView Planting Site

### PR DESCRIPTION
**Pull request by karthikbtf**
- used suggestions from Hannah here https://terraformation.atlassian.net/browse/SW-2421

Terraware App 2022-12-05 08-44-59
Terraware App 2022-12-05 08-44-47
Terraware App 2022-12-05 08-44-33
Terraware App 2022-12-05 08-44-13
Terraware App 2022-12-05 08-43-56
Terraware App 2022-12-05 08-43-40
Terraware App 2022-12-05 08-43-24
Terraware App 2022-12-05 08-43-10

### CODE REVIEW

File: src/components/PlantingSites/PlantingSiteCreate.tsx [hunk 1]
```diff
@@ -127,55 +111,57 @@ export default function CreatePlantingSite(props: CreatePlantingSiteProps): JSX.
...
                 <Typography
                   fontSize={selectedPlantingSite ? '20px' : '24px'}
                   fontWeight={600}
                   margin={theme.spacing(1, 0)}
                 >
                   {record.id !== -1 && selectedPlantingSite ? selectedPlantingSite.name : strings.ADD_PLANTING_SITE}
                 </Typography>
-                {record.id === -1 && (
-                  <Typography fontSize='14px' fontWeight={400} margin={theme.spacing(0, 0, 3, 0)}>
-                    {strings.ADD_PLANTING_SITE_DESCRIPTION}
```
[the remark this answers is not in the document]
Comment (karthikbtf) [reply src/components/PlantingSites/PlantingSiteCreate.tsx:155]: None of the locations (seed  bank, nursery) have it. Planting site will be under locations as well. Went with that consistency. Also the description does not convey anything useful?

[the remark this answers is not in the document]
Comment (constanzauanini) [reply src/components/PlantingSites/PlantingSiteCreate.tsx:155]: Ok, we can check with Chudo/Rachel and we can remove it from the strings file if we are not using it

[the remark this answers is not in the document]
Comment (karthikbtf) [reply src/components/PlantingSites/PlantingSiteCreate.tsx:155]: This is a Hannah decision. We are not showing this for any of the locations. We have been asked to see how other pages are done - prefer to get this in as per that process.

**Review by constanzauanini — APPROVED** [review]

**Comment by hannah-cai** [comment]
the "import boundaries" card shouldn't have a border but aside from that everything looks good!

**Comment by karthikbtf** [comment]
> the "import boundaries" card shouldn't have a border but aside from that everything looks good!

Thanks @hannah-cai i will make this update.

**Pull request merged**